### ISSUE TITLE
Infinite loop with specific crontab

### ISSUE THREAD

**Issue by undsoft** [Hexagon/croner#66]
```js
const cron = new Cron('0 * * * mon,tue,wed,fri,sat,sun', {
  legacyMode: true,
});
cron.next(new Date('2022-03-31 11:40:34'));
```

Results in an infinite loop.

**Comment by Hexagon** [comment]
Solution in https://github.com/Hexagon/croner/commit/bc635272a9a34224920afdbbd8206c1c549707d1

**Comment by Hexagon** [comment]
Fixed in 4.2.3
